packages/dev/core/src/Engines/Extensions/engine.views.ts

```diff
@@ -156,6 +161,67 @@ Engine.prototype.unRegisterView = function (canvas: HTMLCanvasElement): Engine {
     return this;
 };
 
+Engine.prototype._renderViewStep = function (view: EngineView, parent: HTMLCanvasElement): boolean {
+    const canvas = view.target;
+    const context = canvas.getContext("2d");
+    if (!context) {
+        return false;

```

Comment:
You should return `true` instead, else it is a breaking change.

Reply:
good catch